blender/examples/download/main.rs

```diff
@@ -12,5 +14,8 @@ fn main() {
     let blender = manager
         .fetch_blender(&version)
         .expect("Unable to download Blender!");
-    println!("Blender downloaded at: {:?}", blender.get_executable());
+    println!(
+        "Blender downloaded at: {:?}",
+        AsRef::<PathBuf>::as_ref(&blender)
```

Comment:
Not 100% sure but this might be a better use case for Deref coercion instead.

Reply:
After some time thinking this through, I originally wanted to use AsRef trait to access underlying private variable. I think having a separate function that explicitly return the variable I need to access would be more clear and clean than using Deref/AsRef. 

This code here is used as an example. Will be fixed shortly!